go-apps/meep-rnis/server/rnis.go

```diff
@@ -2746,9 +2798,128 @@ func populateL2Meas(key string, jsonInfo string, l2MeasData interface{}) error {
 		cellIndex = len(data.l2Meas.CellInfo) - 1
 	}
 
+	jsonPoaData, _ := rc[RNIS_DB_CONNECTOR_INDEX].JSONGetEntry(baseKey+"POA:"+ueData.ParentPoaName, ".")
+
+	latency := int32(0)
+	poaPacketLoss := int32(0)
+	if jsonPoaData != "" {
+		poaDataObj := convertJsonToPoaInfo(jsonPoaData)
+		if poaDataObj != nil {
+			latency = poaDataObj.Latency
+			ploss := poaDataObj.PacketLoss
+			//return between 10^-4 t 10^-6
```

Comment:
yes 0 is 10^-6 and 100 is 10^-4... which is why we currently always have 100 since the granularity stops at 0.01. QCIs are mainly between 10^-6 and 10^-4, which i guess this is why they used that, but the 0..100 range is limitative. This is from the spec.